src/clarity-angular/data/datagrid/datagrid-cell.ts

```diff
@@ -14,10 +15,48 @@ import { DatagridHideableColumn } from "./datagrid-hideable-column";
     `,
     host: {
         "[class.datagrid-cell]": "true",
-        "[class.datagrid-cell--hidden]": "hidden"
+        "[class.datagrid-cell--hidden]": "hidden",
+        "[class.datagrid-signpost-trigger]": "hasSignpost"
```

Comment:
i think you can directly use signpost like:

`"[class.datagrid-signpost-trigger]": "signpost.length > 0"`

without creating the boolean variable `hasSignpost`.